cmd/miniooni/main.go

```diff
@@ -222,65 +217,43 @@ func main() {
 	}
 
 	name := getopt.Args()[0]
-
-	if name == "web_connectivity" {
-		log.Info("Fetching test lists")
+	builder, err := sess.NewExperimentBuilder(name)
+	if err != nil {
+		log.WithError(err).Fatal("cannot create experiment builder")
+	}
+	if builder.NeedsInput() {
 		if len(globalOptions.inputs) <= 0 {
-			list, err := testlists.NewClient(sess).Do(ctx, sess.ProbeCC(), 100)
+			log.Info("Fetching test lists")
+			config := sess.NewTestListsConfig()
+			config.Limit = 16
+			client := sess.NewTestListsClient()
+			list, err := client.Fetch(config)
```

Comment:
This is a feature that allows you to query test lists from a session, w/o importing more confusing packages